### PR TITLE
feat: created a product table. resolves issue #6

### LINKED ISSUE #3: Reduce items in NFe table header

### LINKED ISSUE #6: Create a product table.

### PR DESCRIPTION
**Pull request by Machado21**
I have created a component to product table, using another page to show it.
It's showing in home page .

### CODE REVIEW

File: src/app/products-tracking/products-tracking.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <thead>
+        <tr>
+            <!-- <th><input type="checkbox" class="check__all"></th> -->
+            <th>Código</th>
+            <th>Descrição</th>
+            <!-- <th>Transportadora</th> -->
+            <th>Data de Saída</th>
+            <th>Quantidade</th>            
+            <th>PIN</th>
+            <!-- <th>Status</th> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion
           
```
remove comment

File: src/app/products-tracking/products-tracking.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        </tr>
+    </thead>
+    <tbody>
+        <tr *ngFor="let prod of products">
+
+            <!-- <th><input type="checkbox" id="{{prod.id}}"></th> -->
+            <td>{{prod.factory_code}}</td>
+            <!-- <td>{{prod.emissionNf | date: 'dd/MM/yyyy'}}</td> -->
+            <td>{{prod.description}}</td>
+            <!-- <td>{{prod.arrival_forecast | date: 'dd/MM/yyyy'}}</td> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion
           
```
remove comment

File: src/app/products-tracking/products-tracking.component.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <!-- <td>{{prod.date_out | date: 'dd/MM/yyyy'}}</td>             -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion
            
```
remove comment

File: src/app/products-tracking/products-tracking.component.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            <!-- <td>{{prod.pin_release | date: 'dd/MM/yyyy'}}</td> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion

```
remove comment

File: src/app/products-tracking/products-tracking.component.less [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+            right: 0;
+            position: absolute;
+            height: 1px;
+            background: #025de6;
+            width: 100%;
+            opacity: 0;
+            visibility: hidden;
+          }
+          &:before {
+            top: -1px;
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.less]: Is it possible to change this to remove negative numbers?

File: src/app/products-tracking/products-tracking.component.less [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+          }
+          &:after {
+            bottom: -1px;
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.less]: Is it possible to change this to remove negative numbers?

File: src/app/products-tracking/products-tracking.component.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
+<table class="tracking__table">
+    <thead>
+        <tr>
+            <!-- <th><input type="checkbox" class="check__all"></th> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion

```
remove comment

File: src/app/products-tracking/products-tracking.component.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+            <th>Código</th>
+            <th>Descrição</th>
+            <!-- <th>Transportadora</th> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion

```
Remove comment

File: src/app/products-tracking/products-tracking.component.html (hunk 1 above)
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion

```
Remove comment

File: src/app/products-tracking/products-tracking.component.html [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+            <th>PIN</th>
+            <!-- <th>Status</th> -->
+        </tr>
+    </thead>
+    <tbody>
+        <tr *ngFor="let prod of products">
+
+            <!-- <th><input type="checkbox" id="{{prod.id}}"></th> -->
+            <td>{{prod.factory_code}}</td>
+            <!-- <td>{{prod.emissionNf | date: 'dd/MM/yyyy'}}</td> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion

```
Remove comment

File: src/app/products-tracking/products-tracking.component.html [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+            <th>Data de Saída</th>
+            <th>Quantidade</th>            
+            <th>PIN</th>
+            <!-- <th>Status</th> -->
+        </tr>
+    </thead>
+    <tbody>
+        <tr *ngFor="let prod of products">
+
+            <!-- <th><input type="checkbox" id="{{prod.id}}"></th> -->
```
Comment (brunorcx) [inline src/app/products-tracking/products-tracking.component.html]:
```suggestion

```
Remove comment

**Review by brunorcx — CHANGES_REQUESTED** [review]

File: src/app/tracking/tracking.component.less
Comment (brunorcx) [inline src/app/tracking/tracking.component.less:1]: Good job!

**Review by brunorcx — APPROVED** [review]

**Comment by brunorcx** [comment]
Good job fixing those requested changes.

**Review by Machado21 — COMMENTED** [review]
fixed

**Pull request merged**